processes/cve_management_process.md

```diff
@@ -0,0 +1,124 @@
+# Node.js CVE management process
+
+The Node.js project acts as a [Commonn Vulnerabilities and Exposures (CVE)
+Numbering Authority (CNA)](https://cve.mitre.org/cve/cna.html).
+The current scope is for all actively developed versions of software
+developed under the Node.js project (ie.  https://github.com/nodejs).
+This means that the Node.js team reviews CVE requests and if appropriate
+assigns CVE numbers to vulnerabilities.  The scope currently **does not**
+include third party modules.
+
+More detailed information about the CNA program is available in
+https://cve.mitre.org/cve/cna/CNA_Rules_v1.1.pdf
+
+# Contacts
+
+As part of the CNA program the Node.js team must provide a number
+of contact points.  Email aliases have been setup for these as follows:
+
+* **Public contact points**. Email address to which people will be directed
+  by Mitre when they are asked for a way to contact the Node.js team about
+  CVE-related issues. **cve-request@iojs.org**
+
+* **Private contact points**. Administrative contacts that Mitre can reach out
+   to directly in case there are issues that require immediate attention.
+   **cve-mitre-contact@iojs.org**
+
+* **Email addresses to add to the CNA email discussion list**. This address has
+   been added to a closed mailing list that is used for announcements,
+   sharing documents, or discussion relevant to the CNA community.
+   The list rarely has more than ten messages a week.
+   **cna-discussion-list@iojs.org** 
+
+# CNA management processes
+
+## CVE Block management
+
+The CNA program allows the Node.js team to request a block of CVE's in
+advance. These CVE's are managed in an issue within the private Node.js
+security repo (https://github.com/nodejs/security).  Each year there
+will be an issue in that repo titled:
+
+```
+CVE Block for XXXX
+```
+
+were XXXX is the year (for example `CVE Block for 2017`).
+
+This issue will have the following sections:
+
+* Available
+* Pending
+* Announced 
+
+
+When a new block of CVEs  is received from Mitre they will be listed under
+the `Available` secion. 
+
+These CVE's will be moved from the Available to Pending and Announced
```

Comment:
CVEs vs. CVE's. Can you use one or the other consistently. I assume it's CVEs since it is not possessive.

Reply:
Fixed